packages/icons/src/library/add-card.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const addCard = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path
+			d="M18.5 5.5V8H20V5.5h2.5V4H20V1.5h-1.5V4H16v1.5h2.5zM12 4H6a2 2 0 00-2 2v12a2 2 0 002 2h12a2 2 0 002-2v-6h-1.5v6a.5.5 0 01-.5.5H6a.5.5 0 01-.5-.5V6a.5.5 0 01.5-.5h6V4z"
+			fillRule="evenodd"
```

Comment:
Code looks good, thanks for flattening the shape. For every time I see these "fillRule" instances, I try to remove them and see if the SVG works without them. Most do, some do require them. I haven't tested whether they are needed here, but just wanted to share that it's a nice thing to test.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32371/commits/cde14654b529a832d84ef67424bb1d31c6399772

For reference, `fillRule` is related to how the shape will be filled and is required for shapes that are closed and contain an empty space inside them. `evenodd` indicates that the outer shape should be filled and that the inside shape shouldn't.

In the case of `addCard` we don't need them, in the case of `bug` and `key` we do need them because they have inner, non-filled shapes.

Reply:
Thank you! I never did delve into the details.